src/main/java/com/epam/test_generator/dto/CaseDTO.java

```diff
@@ -39,10 +39,8 @@
     private Status status;
 
     public CaseDTO() {
-        SimpleDateFormat formatter = new SimpleDateFormat("MM/dd/yyyy");
-        creationDate = formatter.format(Calendar.getInstance().getTime());
-        updateDate = formatter.format(Calendar.getInstance().getTime());
-
+        creationDate = Calendar.getInstance().getTime();
```

Comment:
I'd recommend excluding time to a local variable because if GC is initiated between getTime executions and you'll get different values in creationDate and updateDate.